src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
According to the task checklist, the `get()` method on the service layer should return the entity directly, not an `Optional`. However, the DAO layer should return an `Optional`. Please adjust the `get()` method signature to return `MovieSession` instead of `Optional<MovieSession>`.